.ruff.toml

```diff
@@ -361,6 +366,7 @@ select = [
     "UP038",  # Use `X | Y` in `{}` call instead of `(X, Y)`
     "UP039",  # Unnecessary parentheses after class definition
     "UP040",  # Type alias `{name}` uses `TypeAlias` annotation instead of the `type` keyword
+    "UP041",  # Replace aliased errors with `TimeoutError`
```

Comment:
I don't think it only affects this kind of exception, or does it?

Reply:
That's just a copy from the ruff [Rules](https://docs.astral.sh/ruff/rules/#pyupgrade-up) pages.
> | Code  | Name                                                                            | Message                                    |       | 
> | :---  | :---                                                                            | :---                                       | :---  | 
> | UP041 | [`timeout-error-alias`](https://docs.astral.sh/ruff/rules/timeout-error-alias/) | Replace aliased errors with `TimeoutError` | 🛠️ 🧪 |